<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_export</class>
 <widget class="QDialog" name="Dialog_export">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>505</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Экспорт таблиц</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/logo.png</normaloff>icon/logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="lab1">
       <property name="text">
        <string>Выберите таблицы</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="5">
      <widget class="QTableWidget" name="tableWidget"/>
     </item>
     <item row="2" column="3">
      <widget class="QCheckBox" name="checkBox_selectAll">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Выбрать все</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="lab2">
       <property name="text">
        <string>Выходная папка</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QToolButton" name="toolB_selectWay">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="lab3">
       <property name="text">
        <string>Настройка проекции</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QToolButton" name="toolB_selectCK">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="5">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QLabel" name="lab4">
       <property name="text">
        <string>Экспорт таблиц</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2" colspan="3">
      <widget class="QProgressBar" name="progBar_tab">
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="2">
      <widget class="QLabel" name="lab5">
       <property name="text">
        <string>Конвертация данных</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2" colspan="3">
      <widget class="QProgressBar" name="progBar_data">
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="5">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="11" column="2" colspan="3">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="4">
      <widget class="QLineEdit" name="lineEdit_way"/>
     </item>
     <item row="6" column="0" colspan="4">
      <widget class="QLineEdit" name="lineEdit_CK"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_export</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_export</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
